crates/ruff_python_parser/tests/snapshots/invalid_syntax@invalid_annotation_class.py.snap

```diff
@@ -0,0 +1,479 @@
+                                TypeVar(
+                                    TypeParamTypeVar {
+                                        range: 198..209,
+                                        name: Identifier {
+                                            id: Name("T"),
+                                            range: 198..199,
+                                        },
+                                        bound: Some(
+                                            Named(
+                                                ExprNamed {
+                                                    range: 202..208,
+                                                    target: Name(
+                                                        ExprName {
+                                                            range: 202..203,
+                                                            id: Name("x"),
+                                                            ctx: Store,
+                                                        },
+                                                    ),
+                                                    value: NumberLiteral(
+                                                        ExprNumberLiteral {
+                                                            range: 207..208,
+                                                            value: Int(
+                                                                1,
+                                                            ),
+                                                        },
+                                                    ),
+                                                },
+                                            ),
+                                        ),
+                                        default: None,
+                                    },
+                                ),
+                            ],
+                        },
+                    ),
+                    arguments: None,
+                    body: [
+                        Expr(
+                            StmtExpr {
+                                range: 212..215,
+                                value: EllipsisLiteral(
+                                    ExprEllipsisLiteral {
+                                        range: 212..215,
+                                    },
+                                ),
+                            },
+                        ),
+                    ],
+                },
+            ),
+        ],
+    },
+)
+```
+## Semantic Syntax Errors
+
+  |
+1 | class F[T](y := list): ...
+  |            ^^^^^^^^^ Syntax Error: named expression cannot be used within a generic definition
+2 | class G((yield 1)): ...
+3 | class H((yield from 1)): ...
+  |
+
+
+  |
+1 | class F[T](y := list): ...
+2 | class G((yield 1)): ...
+  |          ^^^^^^^ Syntax Error: yield expression cannot be used within a base class
```

Comment:
The "within a base class" sounds a bit confusing to me, what about "cannot be used as a base class"